package.json

```diff
@@ -29,7 +29,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "^1.74.0-insider"
+        "vscode": "^1.75.0"
```

Comment:
Shouldn't this be 1.75.0-insider

Reply:
No, we decided in Slack that `-insider` doesn't do anything.

Reply:
Oh it doesn't, ok, i ididn't know that.